app/views/admin/users/index.html.erb

```diff
@@ -13,12 +13,12 @@
       <th><%= t('.columns.name') %></th>
       <th><%= t('.columns.email') %></th>
       <th><%= t('.columns.status') %></th>
-      <th><%= t('.columns.last_sign_in') %></th>
+      <th><%= t('.columns.last_sign_in') %><eth>
```

Comment:
My assumption is that this was made in error and should be `/th`?

Reply:
Fixed